hack/generate-manifest.sh

```diff
@@ -32,6 +32,7 @@ Generate a YAML manifest for Antrea using Kustomize and print it to stdout.
         --no-legacy-crd               Generate a manifest without legacy CRD mirroring support enabled
         --endpointslice               Generate a manifest with EndpointSlice support enabled
         --no-np                       Generate a manifest with Antrea-native policies disabled
+        --configmap-only              Generate a antrea-config manifest appropriate for Antrea agent and controller
```

Comment:
```suggestion
        --configmap-only              Generate an antrea-config manifest appropriate for Antrea agent and controller
```

Reply:
No need to rerun tests with only this change.

Reply:
done